### ISSUE TITLE
body.fgColor applies to dialog text instead of process output

### ISSUE THREAD

**Issue by scottjad** [F1bonacc1/process-compose#167]
With the following theme:
```yaml
style:
  body:
    fgColor: green
  dialog:
    fgColor: red
```

I see the green in dialog text and process output remains white. I expected the white text in the screenshot below to be green.

![image](https://github.com/F1bonacc1/process-compose/assets/124373/489f040c-4846-4f17-9f89-8882b3ed6547)

Thanks for the theming support!

**Comment by F1bonacc1** [comment]
Hey @scottjad 
I intentionally left the log text untouched because of the default ANSI colors mode.

I can add log theming as well if the process is configured with `disable_ansi_colors: true`.
What do you think?

**Comment by scottjad** [comment]
@F1bonacc1 I'm not sure.

I'm trying to make a light theme, so having the log text foreground be white is not ideal.

Is there a way to theme just the log text that uses the default text color (isn't colored by the underlying program)?

I'm not particularly familiar with terminal color technical details.

**Comment by F1bonacc1** [comment]
Good catch! Thanks!
Fixed in v1.0.1
